packages/plugins/typescript/vue-apollo/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@graphql-codegen/typescript-vue-apollo",
```

Comment:
Can you align the version regarding to other packages? Also the dependencies should be aligned as well. After that, I'll merge it :)

Reply:
Alright I updated the version, think the other dependencies are up to date as well.